{"info": {"author": "Marc Cerrato", "author_email": "marccerrato@gmail.com", "bugtrack_url": null, "classifiers": ["Development Status :: 2 - Pre-Alpha", "Intended Audience :: Developers", "License :: OSI Approved :: BSD License", "Natural Language :: English", "Programming Language :: Python :: 2", "Programming Language :: Python :: 2.6", "Programming Language :: Python :: 2.7", "Programming Language :: Python :: 3", "Programming Language :: Python :: 3.3"], "description": "===============================\nPython DNS Failover\n===============================\n\n.. image:: https://badge.fury.io/py/python-dns-failover.png\n    :target: http://badge.fury.io/py/python-dns-failover\n    \n.. image:: https://travis-ci.org/marccerrato/python-dns-failover.png?branch=master\n        :target: https://travis-ci.org/marccerrato/python-dns-failover\n\n.. image:: https://pypip.in/d/python-dns-failover/badge.png\n        :target: https://crate.io/packages/python-dns-failover?version=latest\n\n\n\"Python script to automatically update DNS records for a bunch of servers participating in a Round-Robin DNS setup\"\n\n* Free software: BSD license\n* Documentation: http://python-dns-failover.rtfd.org.\n\nThis project is mainly based on `@dotcloud`_'s work on his project `autodnsfailover`_.\n\nThe main difference is that **python-dns-failover** script is aimed to **run in an external\nmachine** while `autodnsfailover`_ is to run in the participating servers themselves.\n\n.. _`@dotcloud`: https://github.com/dotcloud/\n.. _`autodnsfailover`: https://github.com/dotcloud/autodnsfailover\n\nTODO\n--------\n* Documentation\n* Testing\n\n\n\nHistory\n-------\n\n0.1.0 (2013-12-29)\n++++++++++++++++++\n\n* Core functionality.\n* CloudFlare DNS backend.\n* First release on PyPI.", "description_content_type": null, "docs_url": null, "download_url": "UNKNOWN", "downloads": {"last_day": -1, "last_month": -1, "last_week": -1}, "home_page": "https://github.com/marccerrato/python-dns-failover", "keywords": "python-dns-failover", "license": "BSD", "maintainer": null, "maintainer_email": null, "name": "python-dns-failover", "package_url": "https://pypi.org/project/python-dns-failover/", "platform": "UNKNOWN", "project_url": "https://pypi.org/project/python-dns-failover/", "project_urls": {"Download": "UNKNOWN", "Homepage": "https://github.com/marccerrato/python-dns-failover"}, "release_url": "https://pypi.org/project/python-dns-failover/0.1.0/", "requires_dist": null, "requires_python": null, "summary": "\"Python script to automatically update DNS records for a bunch of servers participating in a Round-Robin DNS setup\"", "version": "0.1.0", "yanked": false, "html_description": "<div class=\"project-description\">\n            <a href=\"http://badge.fury.io/py/python-dns-failover\" rel=\"nofollow\"><img alt=\"https://badge.fury.io/py/python-dns-failover.png\" src=\"https://warehouse-camo.ingress.cmh1.psfhosted.org/2b20c7bc89b7b37ad394b087a56ef8f121e06aa8/68747470733a2f2f62616467652e667572792e696f2f70792f707974686f6e2d646e732d6661696c6f7665722e706e67\"></a>\n<a href=\"https://travis-ci.org/marccerrato/python-dns-failover\" rel=\"nofollow\"><img alt=\"https://travis-ci.org/marccerrato/python-dns-failover.png?branch=master\" src=\"https://warehouse-camo.ingress.cmh1.psfhosted.org/3e71ade31b39adff4410290a7fe8204f364cf40c/68747470733a2f2f7472617669732d63692e6f72672f6d6172636365727261746f2f707974686f6e2d646e732d6661696c6f7665722e706e673f6272616e63683d6d6173746572\"></a>\n<a href=\"https://crate.io/packages/python-dns-failover?version=latest\" rel=\"nofollow\"><img alt=\"https://pypip.in/d/python-dns-failover/badge.png\" src=\"https://warehouse-camo.ingress.cmh1.psfhosted.org/3cb3d41b31d3e3dd148c69f6fdae623998b40508/68747470733a2f2f70797069702e696e2f642f707974686f6e2d646e732d6661696c6f7665722f62616467652e706e67\"></a>\n<p>\u201cPython script to automatically update DNS records for a bunch of servers participating in a Round-Robin DNS setup\u201d</p>\n<ul>\n<li>Free software: BSD license</li>\n<li>Documentation: <a href=\"http://python-dns-failover.rtfd.org\" rel=\"nofollow\">http://python-dns-failover.rtfd.org</a>.</li>\n</ul>\n<p>This project is mainly based on <a href=\"https://github.com/dotcloud/\" rel=\"nofollow\">@dotcloud</a>\u2019s work on his project <a href=\"https://github.com/dotcloud/autodnsfailover\" rel=\"nofollow\">autodnsfailover</a>.</p>\n<p>The main difference is that <strong>python-dns-failover</strong> script is aimed to <strong>run in an external\nmachine</strong> while <a href=\"https://github.com/dotcloud/autodnsfailover\" rel=\"nofollow\">autodnsfailover</a> is to run in the participating servers themselves.</p>\n<div id=\"todo\">\n<h2>TODO</h2>\n<ul>\n<li>Documentation</li>\n<li>Testing</li>\n</ul>\n</div>\n<div id=\"history\">\n<h2>History</h2>\n<h2 id=\"id1\"><span class=\"section-subtitle\">0.1.0 (2013-12-29)</span></h2>\n<ul>\n<li>Core functionality.</li>\n<li>CloudFlare DNS backend.</li>\n<li>First release on PyPI.</li>\n</ul>\n</div>\n\n          </div>"}, "last_serial": 956396, "releases": {"0.1.0": [{"comment_text": "", "digests": {"md5": "15995d87808790bfa12552c555378d83", "sha256": "2a0bcc1424542e93e691526ede7fa3b640f66af70806065e88af9fdec82b698d"}, "downloads": -1, "filename": "python-dns-failover-0.1.0.tar.gz", "has_sig": false, "md5_digest": "15995d87808790bfa12552c555378d83", "packagetype": "sdist", "python_version": "source", "requires_python": null, "size": 8158, "upload_time": "2013-12-29T19:10:18", "upload_time_iso_8601": "2013-12-29T19:10:18.539841Z", "url": "https://files.pythonhosted.org/packages/57/f3/83e26136c3425909fc0be99bed33999ac9fd635b9c3d00af822445ea7ced/python-dns-failover-0.1.0.tar.gz", "yanked": false}]}, "urls": [{"comment_text": "", "digests": {"md5": "15995d87808790bfa12552c555378d83", "sha256": "2a0bcc1424542e93e691526ede7fa3b640f66af70806065e88af9fdec82b698d"}, "downloads": -1, "filename": "python-dns-failover-0.1.0.tar.gz", "has_sig": false, "md5_digest": "15995d87808790bfa12552c555378d83", "packagetype": "sdist", "python_version": "source", "requires_python": null, "size": 8158, "upload_time": "2013-12-29T19:10:18", "upload_time_iso_8601": "2013-12-29T19:10:18.539841Z", "url": "https://files.pythonhosted.org/packages/57/f3/83e26136c3425909fc0be99bed33999ac9fd635b9c3d00af822445ea7ced/python-dns-failover-0.1.0.tar.gz", "yanked": false}], "timestamp": "Fri May  8 03:17:58 2020"}